docs/security/trust/content_trust.md

```diff
@@ -0,0 +1,261 @@
+
+| Key                 | Description                                                                                                                                                                                                                                                                                                                                                                         |
+|---------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| root key            | Root of trust for a image tag. When trust is enabled, you create this key once. Then, you can use this key to create repository keys.  |
+| target and snapshot | These two keys are known together as the "repository" key.  When trust is enabled, you create this key when you add a new image repository.  If you have the root key, you can export the repository key and allow other images creator that sign image tags.       |
+| timestamps          | This key applies to a repository. It allows Docker repositories to have freshness security guarantees without requiring periodic content refreshes on the client's side.  Each tag is signed with a repository key and a timestamp key.                                                                                                                |
+
+With the exception of timestamps, all the keys are generated and stored locally
+client-side. The timestamp is generated and stored in a Docker registry.
+
+The following image depicts the various keys and their relationships:
+
+![Trust components](../images/trust_components.png)
+
+>**WARNING**: Loss of the root key is **very difficult** to recover froM.
+>Correcting this loss requires intervention from [Docker
+>Support](https://support.docker.com) to reset the image tag state. This loss
+>also requires **manual intervention** from every consumer that used the tagged
+>image prior to the loss. You should backup this key somewhere safe. 
+
+Make sure you read [Manage keys for trust](/security/trust/trust_key_mng)
+information for details on creating, securing, and backing up your keys.
+
+## Survey typical trusted operations
+
+This section surveys the typical trusted operations users perform with Docker
+images.
+
+### Enable trust
+
+Enable trust by setting the `DOCKER_CONTENT_TRUST` environment variable.
+Enabling per-shell is useful because you can have one shell configured for
+trusted operations and another terminal shell for untrusted operations. You can
+also add this declaration to your shell profile to have it turned on always by
+default.
+
+To enable trust in a `bash` shell enter the following command:
+
+```bash
+export DOCKER_CONTENT_TRUST=1
+```
+
+Once set, each of the "tag" operations require key for trusted tag. All of these
+commands also support the `--disable-content-trust=false` flag. This flag allows
```

Comment:
I'm not sure this is the place for this comment but : ``--disable-content-trust=false`` *naively* means that the content trust is enable (because of the double negation). It should be ``--disable-content-trust=true`` on this, right ?